resources/zib/zib-MultidisciplinaryTeamMeeting.xml

```diff
@@ -0,0 +1,239 @@
+      <slicing>
+        <discriminator>
+          <type value="value" />
+          <path value="url" />
+        </discriminator>
+        <rules value="open" />
+      </slicing>
+      <min value="0" />
+    </element>
+    <element id="Encounter.extension:patientConsultationLabel">
+      <path value="Encounter.extension" />
+      <sliceName value="patientConsultationLabel" />
+      <min value="0" />
+      <max value="1" />
+      <type>
+        <code value="Extension" />
+        <profile value="http://nictiz.nl/fhir/StructureDefinition/ext-MultidisciplinaryTeamMeeting.PatientConsultationLabel" />
+      </type>
+    </element>
+    <element id="Encounter.status">
+      <path value="Encounter.status" />
+      <comment value="This element is implictly mapped to the zib concept NL-CM:15.1.3 (PatientConsultationDateTime), which maps to both `Encounter.period.start` and `Encounter.period.end`. Unless the status is explicitly recorded, the following guidance applies:&#xD;&#xA;&#xD;&#xA;* When PatientConsultationDateTime is in the future, `.status` will usually be set to _planned_.&#xD;&#xA;* When `Encounter.period.end` is present and in the past, `.status` will usually be set to _finished_.&#xD;&#xA;* When PatientConsultationDateTime is in the past and `Encounter.period.end` is either equal to `.start` or not present, this means it was a point-in-time encounter and `.status` will usually be set to _finished_.&#xD;&#xA;* When a system is unable to infer the status from the PatientConsultationDateTime, `.status` will be set to _unknown_.&#xD;&#xA;&#xD;&#xA;The _unknown_ code is not to be used to convey other statuses.  The _unknown_ code should be used when one of the statuses applies, but the authoring system doesn't know the current state of the encounter.&#xA;&#xA;This element is labeled as a modifier because the status contains codes that mark the resource as not currently valid." />
+      <mapping>
+        <identity value="zib-multidisciplinaryteammeeting-v1.0-2020EN" />
+        <map value="NL-CM:15.1.3" />
+        <comment value="PatientConsultationDateTime (implicit, main mapping is on `Encounter.period`)" />
+      </mapping>
+    </element>
+    <element id="Encounter.type">
+      <path value="Encounter.type" />
+      <slicing>
+        <discriminator>
+          <type value="pattern" />
+          <path value="$this" />
+        </discriminator>
+        <rules value="open" />
+      </slicing>
+      <min value="1" />
+    </element>
+    <element id="Encounter.type:multidisciplinaryTeamMeetingCode">
+      <path value="Encounter.type" />
+      <sliceName value="multidisciplinaryTeamMeetingCode" />
+      <min value="1" />
+      <max value="1" />
+      <patternCodeableConcept>
+        <coding>
+          <system value="http://snomed.info/sct" />
+          <code value="384682003" />
+        </coding>
+      </patternCodeableConcept>
+    </element>
+    <element id="Encounter.subject">
```

Comment:
How to deal with a patient group as a subject versus a single patient subject in this resource? Is it the expectation that it is always a single patient? Considering the fact that there's also a reason reference and a reference from careplan to this encounter, which are both related to a single patient. So it would be strange to have a patient group as a subject then. And if you want to show these meetings in the health care record or personal environment of the patient it is also easier to have it related to single patients.
On the other hand, I can imagine there are also scenario's in which you would like to define a group as the subject of a multidisciplinary team meeting, e.g. from a healthcare providers perspective, for the planning and preparation of these meetings.

Reply:
The zib is defined in terms of 'a/the patient', so I guess this question would be better aimed at the zib-centrum with the use case you mention. 

From a FHIR perspective the Group resource could be used here. We are not limiting the use of that resource for specific use cases.

Reply:
Thanks for your review by the way!

Reply:
No problem, we need this profile for birthcare as well, so I copied most of your work in a STU3 version of this profile. So, thank you for your work as well!